src/core.js

```diff
@@ -172,6 +172,7 @@ extend( QUnit, {
 } );
 
 QUnit.pushFailure = pushFailure;
+QUnit.pushFailureAndResume = pushFailureAndResume;
```

Comment:
I've noted this in the main conversation, but I'm using QUnit.failTest() at this point.